syncto/views/record.py

```diff
@@ -30,3 +30,16 @@ def record_get(request):
     handle_headers_conversion(sync_client.raw_resp, request.response)
 
     return {'data': record}
+
+
+@record.delete(permission=NO_PERMISSION_REQUIRED)
+def record_delete(request):
+    collection_name = request.matchdict['collection_name']
+    record_id = request.matchdict['record_id']
+    sync_id = uuid4_to_base64(record_id)
+
+    sync_client = build_sync_client(request)
+    sync_client.delete_record(collection_name, sync_id)
+
+    request.response.status_code = 204
```

Comment:
Cool. Makes me wonder if we should return a SyncException instead (in syncclient). Thoughts?